Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -188,6 +190,10 @@ private async Task<bool> CanReadOneAsync(string resource)
                 }
             case Resource.NEWS:
                 return true;
+            case Resource.STUDENT_SCHEDULE:
+                if (context.ActionArguments["username"] is string viewed_username)
+                    return user_groups.Contains(AuthGroup.Advisors) || viewed_username.EqualsIgnoreCase(user_name) || _accountService.GetAccountByUsername(viewed_username).AccountType.EqualsIgnoreCase("FACULTY");
```

Comment:
I am not positive that `ACCOUNT.AccountType == "FACULTY"` will align perfectly with the `profile.PersonType.includes('fac')` check that we do on the front end to decide whether to show a person's schedule to another user. That's because `Profile.PersonType` is calculated based on whether a row was discovered in `CCT.dbo.FacultyStaff` for the given user.

This is a pretty grey concern though, and won't matter once the new code is in, so it should be fine to push through in this hotfix.